spec/intl.html

```diff
@@ -2123,14 +2106,26 @@ <h1>Temporal.Calendar.prototype.mergeFields ( _fields_, _additionalFields_ )</h1
             1. Let _calendar_ be the *this* value.
             1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
             1. Set _fields_ to ? ToObject(_fields_).
+            1. Let _merged_ be OrdinaryObjectCreate(%Object.prototype%).
+            1. Perform ? CopyDataProperties(_merged_, _fields_, « », « *undefined* »).
             1. Set _additionalFields_ to ? ToObject(_additionalFields_).
-            1. If _calendar_.[[Identifier]] is *"iso8601"*, then
-              1. Return ? DefaultMergeCalendarFields(_fields_, _additionalFields_).
-            1. Let _fieldsCopy_ be OrdinaryObjectCreate(%Object.prototype%).
-            1. Perform ? CopyDataProperties(_fieldsCopy_, _fields_, « », « *undefined* »).
-            1. Let _additionalFieldsCopy_ be OrdinaryObjectCreate(%Object.prototype%).
+            1. Let _additionalFieldsCopy_ be OrdinaryObjectCreate(*null*).
             1. Perform ? CopyDataProperties(_additionalFieldsCopy_, _additionalFields_, « », « *undefined* »).
-            1. Return CalendarDateMergeFields(_calendar_.[[Identifier]], _fieldsCopy_, _additionalFieldsCopy_).
+            1. NOTE: Every property of _additionalFieldsCopy_ is a data property with non-*undefined* value, but some property keys may be Symbols.
+            1. Let _additionalKeys_ be ! _additionalFieldsCopy_.[[OwnPropertyKeys]]().
+            1. For each element _key_ of _additionalKeys_, do
+              1. If _calendar_.[[Identifier]] is *"iso8601"*, then
+                1. If _key_ is *"month"*, let _excludedKeys_ be « *"monthCode"* ».
+                1. Else if _key_ is *"monthCode"*, let _excludedKeys_ be « *"month"* ».
+                1. Else let _excludedKeys_ be « ».
+              1. Else,
+                1. Let _excludedKeys_ be the List of property keys that are mutually excluded by _key_.
```

Comment:
Do you want to have a bit more context about what "mutually excluded" means? The original text was IMO a bit clearer about this.

Reply:
+1, i really have no idea how i'd implement this line. can we use explicit algorithm steps?

Reply:
We could pull this part out into a separate op or whatever, but I'd like to resolve the discussion in #2407 whether this is the approach we want before doing any fine-tuning of the language in this PR.

Reply:
I've pulled the calendar-dependent part out into a separate operation with an explanation of "mutually excluded" and examples.